module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -415,6 +415,7 @@ public function getInputFilter(): InputFilterInterface
             ]
         ];
 
+        $elementObjects = parent::getElements();
```

Comment:
Is this a sign that we need to rename our getElements() method to avoid a naming conflict, or are we intentionally overriding the existing one? It's a bit unclear why we would do things this way -- if it really is intentional, perhaps a comment to explain why would help prevent future confusion.

Reply:
I'd support renaming to avoid total confusion. At least I've already stumbled on this a couple of times.

Reply:
I agree that this is very confusing and should be changed. We have some plans to use the form classes in a way that probably would require refactoring the form system anyway, so the question is should this wait until then, or would it be ok to commit this with the clarifying TODO comment?

Reply:
I don't think that the changes here make the existing situation significantly worse -- they just make it more obvious that a problem exists. Thus, I'm comfortable with merging them and addressing the rest separately. The question is whether it's worth doing a quick fix for 8.0 now, or waiting longer and doing a bigger refactor later.